app/controllers/application_controller.rb

```diff
@@ -4,10 +4,28 @@
 class ApplicationController < ActionController::Base
   include Clearance::Controller
   protect_from_forgery with: :exception
+  before_action :filter_by_company, if: :company_subdomain?
 
   private
 
   def current_company
     current_user.company
   end
+
+  def filter_by_company
+    @filter_by = matching_company || not_found
```

Comment:
just a 💅 I suggested using `or` instead of `||` to work as a flow control, even `||` working properly, I **think (not 100% sure though)** that `or` might suit better in this case. Need some backup here 🤣, @duduribeiro.

Reply:
Yes! You are right @fpgentil 👍 . `Or` is the perfect case for this. Because it is flow logic and not a boolean operation.

Reply:
Well I agree with you on this, in fact I've pushed this code with `or` at first, but `ebert` complained asking me to change it to `||`, and I did. Should I just ignore ebert or there is a proper way to handle this?

Reply:
I think you can ignore ebert. Try changing it and run `rubocop` to check if it will not complain about it. Or is the best option here imho

Reply:
It seems like `rubocop` doesn't agree with us:

```
➜  opensanca_jobs git:(handle-subdomain) ✗ rubocop
NOTE: Gem.gunzip is deprecated; use Gem::Util.gunzip instead. It will be removed on or after 2018-12-01.
Gem.gunzip called from /usr/lib/ruby/gems/2.5.0/gems/unicode-display_width-1.3.0/lib/unicode/display_width/index.rb:5.
Inspecting 45 files
............................C................

Offenses:

app/controllers/application_controller.rb:16:35: C: Style/AndOr: Use || instead of or.
    @filter_by = matching_company or not_found
                                  ^^
```

I'm going to keep using `||` then. If any of you guys knows about a better solution, I'm all ears.